spec/index.html

```diff
@@ -442,42 +444,43 @@ <h3>Quoted Literals</h3>
         <a href="#grammar-production-RDFLiteral">RDFLiteral</a>) have a lexical form
         followed by a language tag, a datatype IRI, or neither.
         The representation of the lexical form consists of an initial delimiter
-        (e.g., <code>"</code> (<span class="codepoint">U+0022</span>));
+        (e.g., '<code>&quot;</code>' (quotation mark, code point <code class="codepoint">U+0022</code>));
```

Comment:
Having doubles quotes include single quotes looks confusing. It's already a different color.

Reply:
> I note you did not raise this concern

Do not take that as agreement elsewhere.

Reply:
I am thinking we may need a few samples (i.e., several copies of one or two of the paragraphs in question, each copy with a different wrapper or lack thereof) to get merged into this or another PR, so PR Preview can show them as they would be after going through ReSpec, and we can then make a decision as a group.